summaries/sprint2.md

```diff
@@ -0,0 +1,27 @@
+# Week 2 sprint retrospective
+
+## Scrum master (Konrad, Jonas)
+This week we vastly improved on time management in meetings. They were much shorter and a lot more efficient.
+We had two short standups (Monday and Thursday) during which everyone updated the group on their current status and their dependencies on work from other people. After the meeting these dependencies where resolved by the concerned parties.
+Average meeting duration was 15 minutes.
+Pull request were there faster but not entirely ready for merging on Wednesday because of troubles writing the tests.
+
+## Bettens, Louis
+This week I provided a well-modularised implementation of the cloud database back-end of the app. This required discussing the interface of the module with the rest of the team, in particular with Martin who provided it. I also had to fight the CI to get acceptable test coverage on my work due to odd issues with the Glide library, which took a lot of time at the end of the process and almost caused a failure to deliver on time.
+
+I noted that, since Martin and I miscommunicated, it might be preferable in the future to structure such discussions around trying to write down a decision in an electronic document, so as to make sure everything is clear and can be referred back to.
+
+## Jaakik, Marouane
+
+## Konrad, Jonas Franz
+This week I implemented the AddPictureActivity completely. I had to resolve how this activity would communicate with the rest of the app and how it will return the result (the picture which was taken and the assigned category). This was done using an ActivityResultContract.
+I could not find out how to deny permissions during testing which means that I can't test the activity properly. These problems during testing led my pull request to only exit the draft phase on thursday evening, which is suboptimal.
+
+## Lachat, Niels Marco
+

```

Comment:
```suggestion
This week I added unit tests for the learning UI that I implemented in the previous week. I had to use UI Automator because espresso doesn't have an easy way to perform a drag and drop operation. This meant that I had to learn how to use UI Automator. I didn't have the time to integrate the learning UI with the dummy dataset, but it will probably be quite simple to integrate. This week's workload was a bit lighter for me because I was unavailable for two days (I had to go to the recruitment), but I will try to compensate this in next week's sprint.

```